cmake/projects/eyalroz_printf/hunter.cmake

```diff
@@ -0,0 +1,30 @@
+# Copyright (c) 2023 Alexander Voronov
+# All rights reserved.
+
+# !!! DO NOT PLACE HEADER GUARDS HERE !!!
+
+include(hunter_add_version)
+include(hunter_cacheable)
+include(hunter_cmake_args)
+include(hunter_download)
+include(hunter_pick_scheme)
+include(hunter_report_broken_package)
```

Comment:
Please remove unused imports

Reply:
as far as I can see `hunter_report_broken_package` and `hunter_status_debug` are redundant